### PR TITLE
Initial checkin with stub for video

### PR DESCRIPTION
**Pull request by aleffert**
Umbrella configuration document for setting up mobile apps.

Still needs a little work + help on video content from @nasthagiri

### CODE REVIEW

File: en_us/install_operations/source/mobile.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+- Android http://github.com/edx/edx-app-android
+
+**************
+Authentication
+**************
+
+The edX mobile apps require your edX installation to adjust your ``edx-platform`` configuration settings to enable mobile API support.
+
+You will need to add the following under the features section:
```
Comment (nasthagiri) [inline en_us/install_operations/source/mobile.rst]: specify lms.env.json (as opposed to the corresponding CMS environment file).

**Comment by nasthagiri** [comment]
@aleffert I've added the video part of the documentation.

**Comment by aleffert** [comment]
@lamagnifica @mhoeber Can you take a look at this? Thanks.

File: en_us/install_operations/source/mobile.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+.. include:: links.rst
+
+.. _Setting up the Mobile Applications:
+
+######################################
+Setting up the edX Mobile Applications
+######################################
+
+This chapter is intended for those are who are building the edX mobile applications customizing their Open edX installation to support their use.
```
Comment (mhoeber) [inline en_us/install_operations/source/mobile.rst]: "applications AND customizing . . ."

File: en_us/install_operations/source/mobile.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+.. code-block:: json
+
+    "FEATURES" : {
+        ...
+        "ENABLE_MOBILE_REST_API": true,
+        "ENABLE_OAUTH2_PROVIDER": true,
+        ...
+    }
+
+You also need to set the following configuration value at the top level:
```
Comment (mhoeber) [inline en_us/install_operations/source/mobile.rst]: can we be more specific as to what "top level" means?

**Comment by mhoeber** [comment]
@aleffert  looks good, a couple of small comments.  Thanks

**Comment by aleffert** [comment]
@mhoeber @lamagnifica anything else before I merge?

**Comment by mhoeber** [comment]
@aleffert  nope, just squash commits

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @aleffert! I've created [OSPR-5350](https://openedx.atlassian.net/browse/OSPR-5350) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by nedbat** [comment]
Hi, sorry for the new comment on this old pull request.  We're testing out re-scanning contributions, and our bot added a confusing message that you can ignore.  We'll fix that. Sorry about the noise. (Hi Akiva!)

**Pull request merged**